core/keyserver/keyserver.go

```diff
@@ -116,9 +116,10 @@ func (s *Server) getEntry(ctx context.Context, domainID, userID, appID string, f
 		glog.Errorf("tlog.GetLatestSignedLogRoot(%v): %v", domain.LogID, err)
 		return nil, grpc.Errorf(codes.Internal, "Cannot fetch SignedLogRoot")
 	}
-	// Use the log as the athoritative source of the latest revision.
+	// Use the log as the authoritative source of the latest revision.
```

Comment:
Please add a TODO to add a similar GetEntryByRevision as well.  You can assign it to me.